sdks/java/io/cassandra/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
+	license agreements. See the NOTICE file distributed with this work for additional 
+	information regarding copyright ownership. The ASF licenses this file to 
+	You under the Apache License, Version 2.0 (the "License"); you may not use 
+	this file except in compliance with the License. You may obtain a copy of 
+	the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required 
+	by applicable law or agreed to in writing, software distributed under the 
+	License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS 
+	OF ANY KIND, either express or implied. See the License for the specific 
+	language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.beam</groupId>
+		<artifactId>beam-sdks-java-io-parent</artifactId>
+		<version>0.6.0-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<artifactId>beam-sdks-java-io-cassandra</artifactId>
+	<name>Apache Beam :: SDKs :: Java :: IO :: Cassandra</name>
+	<description>IO to read and write on Cassandra datasource.</description>
+
+	<properties>
+		<cassandra-driver.version>3.1.4</cassandra-driver.version>
+		<netty.version>4.0.30.Final</netty.version>
+		<guava.version>16.0.1</guava.version>
```

Comment:
FYI this *just* changed in the Java-Driver to support v20: https://github.com/datastax/java-driver/commit/1ddd09eeb92bb381223e081fd691bc93c3ffa4fc and https://datastax-oss.atlassian.net/browse/JAVA-1328